test/evergreen.yml

```diff
@@ -1915,7 +1915,7 @@ tasks:
       - func: "compile wiredtiger"
       - func: "format test script"
         vars:
-          format_test_script_args: -t 110 -j 4 direct_io=1
+          format_test_script_args: -t 110 direct_io=1
```

Comment:
I would update the comment above to say something like "This test is run on a higher spec distro due to historical issues with timeout failures. Consider reducing the parallelism if frequent timeouts occur."

How does that sound? In any case, we need to update the comment to remove the `(x4)`.

Reply:
Yeah, that makes sense, just added it now.